docker/docker-compose.adversary.yml

```diff
@@ -34,7 +34,10 @@ services:
       BLOCKCHAIN_PORT: 8546
       HASH_TYPE: poseidon
       LOG_LEVEL: debug
-      IS_CHALLENGER: 'true'
+      # right now it doesn't matter what is IS_CHALLENGER flag's value is because
+      # optimist container state-sync code start challenges at time of startup
```

Comment:
User's who read this might not be aware, which optimist we are referring to. 
Because during ping pong tests we have adversary optimist and normal optimist.

But this docker compose can be used anytime we want adversary too and not just ping pong.
Can we just specify that we shouldn't expect a bad proposer to be a good challenger.
Hence while using adversary, one needs to up a different container for good optimist for challenging.

Reply:
explanation added as asked for